grr/server/grr_response_server/gui/api_integration_tests/osquery_test.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+# Lint as: python3
+"""Integration tests for the Osquery flow, its api client and api endpoints."""
+from grr_response_server.gui import api_integration_test_lib
+from grr.test_lib import osquery_test_lib
+from grr.test_lib import flow_test_lib
+from grr.test_lib import test_lib
+from grr.test_lib import action_mocks
+
+from grr_response_server.flows.general import osquery as osquery_flow
+
+from grr_response_client.client_actions import osquery as osquery_action
+
+from grr_response_proto.api import osquery_pb2 as api_osquery_pb2
+
+from grr_api_client import utils
```

Comment:
I would not put so many blank lines. I guess you tried to group by package but the convention within our codebase is grouping into 3 groups:

  * standard library imports
  * third-party imports
  * GRR imports